src/ui/pages/History/AccelerateTransactionDialog/CancelTx/CancelTx.tsx

```diff
@@ -205,11 +207,14 @@ function CancelTxContent({
             >
               Back
             </Button>
-            <SignTransactionButton
-              wallet={wallet}
-              ref={signTxBtnRef}
-              onClick={() => sendTransaction()}
-            />
+            {preferences ? (
```

Comment:
I didn't see any jumps in UI because of not rendering the button. But I'm not 100% sure this is the base option here.